docs/rules/init-declarations.md

```diff
@@ -111,10 +115,14 @@ function foo() {
 }
 ```
 
-With `"ignoreForLoopInit"` enabled, the following pattern is not considered a problem.
+The `"never"` option ignores `const` variable initializations.
+
+### ignoreForLoopInit
+
+Examples of **correct** code for the `"never", {"ignoreForLoopInit": true}` options:
 
 ```js
-/*eslint init-declarations: [2, "never", { "ignoreForLoopInit": true }]*/
+/*eslint init-declarations: [2, "never", {"ignoreForLoopInit": true}]*/
```

Comment:
@IanVS A quick search seemed to show 60% without space to 40% with spaces. Let me search again more carefully to make sure about that. With that ratio it is as easy to add as to delete.